### PR TITLE
Add router to app

### PR DESCRIPTION
**Pull request by judy0ye**
What I did: Add router to app

Did this break anything?

- [x] No
- [ ]  Yes

Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:

- [ ]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [ ]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

Notes:

## Contributors to this pull request:
Scotty Brown

### CODE REVIEW

File: src/Components/Movies/Movies.js [hunk 1]
```diff
@@ -1,29 +1,25 @@
...
-  })
-}
-  return (
-    <div className='movies-container'>
-      {movieCards}
-    </div>
-  )
+function Movies({ movies, showSingleMovie }) {
+  const movieCards = movies.movies
+    ? movies.movies.map((movie) => {
```
Comment (Scotty-Brown) [inline src/Components/Movies/Movies.js:7]: Great job refactoring the previous 'if statement' to a turnary, looks great

File: src/Components/SingleMovie/SingleMovie.js [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
-        budget={movie.movie.budget}
-        revenue={movie.movie.revenue}
-        runTime={movie.movie.runtime}
-        overview={movie.movie.overview}
-        goBack={goBack}
-        displayVideo={displayVideo}
-      />
-  )
+function SingleMovie({ movie, goBack, displayVideo }) {
+  return movie ? (
```
Comment (Scotty-Brown) [inline src/Components/SingleMovie/SingleMovie.js:6]: very clean

File: src/Components/App/App.js [hunk 3]
```diff
@@ -41,8 +42,10 @@ function App() {
     <main>
       <Header />
       {error && <ErrorHandling error={error}/>}
-      {!singleMovie ? <Movies movies={movies} showSingleMovie={showSingleMovie} />
-       : <SingleMovie movie={singleMovie} goBack={goBack} displayVideo={displayVideo}/>}
+      <Routes>
```
Comment (Scotty-Brown) [inline src/Components/App/App.js:45]: Routes look great!

File: src/Components/Movies/Movies.js (hunk 1 above)
Comment (nicktu12) [reply src/Components/Movies/Movies.js:7]: since the false option of this ternary is null, consider doing `movies.movies &&` instead of your ternary

**Pull request merged**